fetch/src/1_fetch.r

```diff
@@ -0,0 +1,17 @@
+
+  library(dplyr)
+  library(readr)
+  library(stringr)
+  library(sbtools)
+  library(whisker)
+  
+  fetch_data <- function() {
+    project_output_dir <- 'fetch/out/my_dir'
```

Comment:
Sorry I didn't catch this before, but let's drop the generic `my_dir` naming, and instead send everything to `1_fetch/out` (don't forget to include the step number) or `2_process/out`